From ab4e805f7114b1f85a0d1da144443f76dcaa04a5 Mon Sep 17 00:00:00 2001
From: Konstantin Porotchkin <kostap@marvell.com>
Date: Mon, 3 May 2021 14:25:02 +0300
Subject: [PATCH 1521/1921] arm64: dts: marvell: enable crypto devices on
 cn913x

Enable first crypro device on CN913x platforms.

Signed-off-by: Konstantin Porotchkin <kostap@marvell.com>
Change-Id: I2ee8ecca5428f756f459fbc8167b9e5e8e84e366
Reviewed-on: https://sj1git1.cavium.com/c/IP/SW/kernel/linux/+/50999
Tested-by: sa_ip-sw-jenkins <sa_ip-sw-jenkins@marvell.com>
Reviewed-by: Stefan Chulski <stefanc@marvell.com>
[WK: The original patch got from Marvell sdk11.21.09]
Signed-off-by: Wenlin Kang <wenlin.kang@windriver.com>
---
 arch/arm64/boot/dts/marvell/cn9130-crb.dtsi | 1 -
 arch/arm64/boot/dts/marvell/cn9130-db.dtsi  | 2 +-
 2 files changed, 1 insertion(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/marvell/cn9130-crb.dtsi b/arch/arm64/boot/dts/marvell/cn9130-crb.dtsi
index 45eab1b63d7b..11059d7dca2f 100644
--- a/arch/arm64/boot/dts/marvell/cn9130-crb.dtsi
+++ b/arch/arm64/boot/dts/marvell/cn9130-crb.dtsi
@@ -88,7 +88,6 @@
 };
 
 &cp0_crypto {
-	cell-index = <0x0>;
 	status = "okay";
 };
 
diff --git a/arch/arm64/boot/dts/marvell/cn9130-db.dtsi b/arch/arm64/boot/dts/marvell/cn9130-db.dtsi
index d3d4152dc0f2..63d4229da819 100644
--- a/arch/arm64/boot/dts/marvell/cn9130-db.dtsi
+++ b/arch/arm64/boot/dts/marvell/cn9130-db.dtsi
@@ -118,7 +118,7 @@
 };
 
 &cp0_crypto {
-	status = "disabled";
+	status = "okay";
 };
 
 &cp0_ethernet {
-- 
2.31.1

